src/main/resources/templates/db_scripts/populateEffectiveCallNumberComponentsForExistingItems.sql

```diff
@@ -0,0 +1,25 @@
+UPDATE ${myuniversity}_${mymodule}.item AS it
+SET jsonb =
+  CASE
+    WHEN it.jsonb->'itemLevelCallNumber' IS NOT NULL
+      THEN
+        CASE
+          WHEN it.jsonb->'effectiveCallNumberComponents' IS NULL THEN
+            jsonb_set(it.jsonb, '{effectiveCallNumberComponents}',
+              jsonb_set('{"callNumber":null}', '{callNumber}', it.jsonb->'itemLevelCallNumber'))
```

Comment:
There is also `json_build_object` function, that can be used to build an object:


Example | Example Result
-- | --
json_build_object('foo',1,'bar',2) | {"foo": 1, "bar": 2}